contracts/proxy/Clones.sol

```diff
@@ -4,7 +4,7 @@
 pragma solidity ^0.8.20;
 
 /**
- * @dev https://eips.ethereum.org/EIPS/eip-1167[ERC-1167] is a standard for
+ * @dev https://eips.ethereum.org/EIPS/eip-1167 [ERC-1167] is a standard for
```

Comment:
I think the space here is an typo
```suggestion
 * @dev https://eips.ethereum.org/EIPS/eip-1167[ERC-1167] is a standard for
```

Reply:
It is to fix the link

Reply:
> It is to fix the link

Do you mean in VS Code? This is how it works for the documentation in AsciiDoc, using [URL Macros](https://docs.asciidoctor.org/asciidoc/latest/macros/url-macro/)

Reply:
ohhh, majes sense